src/bundle/Command/MigrateNamespacesCommand.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Bundle\FieldTypeRichText\Command;
+
+use Ibexa\Contracts\Core\Repository\PermissionResolver;
+use Ibexa\Contracts\Core\Repository\UserService;
+use Ibexa\FieldTypeRichText\Persistence\Legacy\ContentModelGateway as Gateway;
+use Symfony\Component\Console\Exception\RuntimeException;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class MigrateNamespacesCommand extends AbstractMultiProcessComand
+{
+    private Gateway $gateway;
+
+    private ?int $cursorStart;
+
+    private ?int $cursorStop;
+
+    private ?int $objectCount;
+
+    public function __construct(
+        PermissionResolver $permissionResolver,
+        UserService $userService,
+        Gateway $gateway
+    ) {
+        $this->objectCount = null;
+        parent::__construct('ibexa:migrate:richtext-namespaces', $permissionResolver, $userService);
+        $this->gateway = $gateway;
+    }
```

Comment:
In general, you want to put command name as a static property, so the command service does not need to be initialized when presenting command lists to the user (`php bin/console list`). This effectively makes the command "lazy".

```suggestion
final class MigrateNamespacesCommand extends AbstractMultiProcessComand
{
    protected static $defaultName = 'ibexa:migrate:richtext-namespaces';

    private Gateway $gateway;

    private ?int $cursorStart;

    private ?int $cursorStop;

    private ?int $objectCount;

    public function __construct(
        PermissionResolver $permissionResolver,
        UserService $userService,
        Gateway $gateway
    ) {
        $this->objectCount = null;
        parent::__construct(null, $permissionResolver, $userService);
        $this->gateway = $gateway;
    }
```

Reply:
I assume your point is to use null in first parameter when calling parent constructor and calling `setName()` inside `configure()` instead. 

Fixed in bf273e402a2049784d8dfb008b58c1ec8b6a2619